PROJECT

```diff
@@ -1,5 +1,5 @@
 version: "2"
-domain: bootstrap.cluster.sigs.k8s.io
+domain: bootstrap.cluster.x-k8s.io
```

Comment:
This should be `cluster.x-k8s.io`

Reply:
Why?

Reply:
ohh the *group* should be bootstrap?

Yeah, the way i have it now there is a bit of redundancy.